src/frontend/staff-portal/src/app/components/staff-workbench/update-request-inbox/update-request-inbox.component.ts

```diff
@@ -0,0 +1,154 @@
+import { Component, OnInit, ViewChild, AfterViewInit, Output, EventEmitter } from '@angular/core';
+import { MatTableDataSource } from '@angular/material/table';
+import { MatSort } from '@angular/material/sort';
+import { DisputeService, Dispute } from 'app/services/dispute.service';
+import { DisputeCountRequestCourtAppearance, DisputeStatus, OcrViolationTicket, Field } from 'app/api';
+import { LoggerService } from '@core/services/logger.service';
+import { Subscription } from 'rxjs';
+import { AuthService, KeycloakProfile } from 'app/services/auth.service';
+
+@Component({
+  selector: 'app-update-request-inbox',
+  templateUrl: './update-request-inbox.component.html',
+  styleUrls: ['../../../app.component.scss', './update-request-inbox.component.scss'],
+})
+export class UpdateRequestInboxComponent implements OnInit, AfterViewInit {
+  @Output() public disputeInfo: EventEmitter<Dispute> = new EventEmitter();
+
+  dataSource = new MatTableDataSource();
+  busy: Subscription;
+  displayedColumns: string[] = [
+    '__RedGreenAlert',
+    '__DateSubmitted',
+    'ticketNumber',
+    'disputantSurname',
+    'disputantGivenNames',
+    'status',
+    '__FilingDate',
+    '__CourtHearing',
+     'userAssignedTo',
+  ];
+  disputes: Dispute[] = [];
+  public userProfile: KeycloakProfile = {};
+  public RequestCourtAppearance = DisputeCountRequestCourtAppearance;
+
+  @ViewChild('tickTbSort') tickTbSort = new MatSort();
+  public showTicket = false
+
+  constructor(
+    public disputeService: DisputeService,
+    private logger: LoggerService,
+    private authService: AuthService,
+  ) {
+    this.disputeService.refreshDisputes.subscribe(x => {this.getAllDisputesWithPendingUpdates();})
+  }
+
+  public async ngOnInit() {
+    this.authService.userProfile$.subscribe(userProfile => {
+      if (userProfile) {
+        this.userProfile = userProfile;
+      }
+    })
+
+    // when authentication token available, get data
+    this.getAllDisputesWithPendingUpdates();
+  }
+
+  isNew(d: Dispute): boolean {
+    return d.status == DisputeStatus.New && (d.emailAddressVerified === true || !d.emailAddress);
+  }
+
+  getAllDisputesWithPendingUpdates(): void {
+    this.logger.log('UpdateRequestInboxComponent::getAllDisputesWithPendingUpdates');
+
+    this.disputes = [];
+
+    this.dataSource.data = this.disputes;
+
+    // initially sort data by Date Submitted
+    this.dataSource.data = this.dataSource.data.sort((a: Dispute, b: Dispute) => { if (a.__DateSubmitted > b.__DateSubmitted) { return -1; } else { return 1 } });
+
+    // this section allows filtering only on ticket number or partial ticket number by setting the filter predicate
+    this.dataSource.filterPredicate = function (record: Dispute, filter) {
+      return record.ticketNumber.toLocaleLowerCase().indexOf(filter.toLocaleLowerCase()) > -1;
+    }
+
```

Comment:
remove lines 68-75

Reply:
nope, left the filter in there

Reply:
since line 64 set it to an empty array and lines 116 - 121 are setting the filter too

Reply:
since line 64 set it to an empty array and lines 116 - 121 are setting the filter too